client/stylesheets/_toCat.scss

```diff
@@ -2148,7 +2149,7 @@ i:hover {
 }
 
 .contactButton {
-    padding: 15px 22px;
+    padding: 13px 18px;
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Properties should be ordered background-color, border, border-radius, box-shadow, color, float, font-size, font-size, overflow, padding